### ISSUE TITLE
iOS 13 dark mode support

### ISSUE THREAD

**Issue by eutampieri** [zipleen/tvheadend-iphone-client#259]
I’m astonished no one else has thought to open this issue.
That said it would be nice if you implemented the dark mode in your application.
Thank you

**Comment by zipleen** [comment]
Hey, that's because this was already discussed a lot in Gitter :p

In order to support the native dark mode I need to remove support for iOS version 9, 10 and 11.
This is now not a big deal anymore but at the time it was.
Now I have another bigger issue.. my main dev machine is still in High Sierra and can't be easily updated - this means Xcode is giving me very big issues because the latest version really doesn't work at all. And VLCkit destroyed the compilation for older versions of Xcode for tvOS builds! 

I need to fix a lot of these stupid little things before I can deprecate older versions... also, once I deprecate them, a lot of older iPads will stop getting updates! 

In a nutshell, I'm being lazy :D blame VR :p

**Comment by eutampieri** [comment]
Why should you remove iOS 9-11 support? I thought you could conditionally choose colours with compiler flags (you can see which OS you’re running on)

Eugenio Tampieri

**Comment by zipleen** [comment]
If you want to support native, as far as I remember you're forced to upgrade the sdk version, which breaks older versions.

Didn't you notice that a lot of apps are now iOS 12 only compatible? That's dark mode support for you :p

**Comment by eutampieri** [comment]
That’s strange… https://github.com/eutampieri/Tsuikaban-iOS/blob/698ef429831a87a49186a5e9ddacf4beb0c47ce9/Tsuikaban/LevelsViewController.swift#L59 here I try to do dark mode without losing iOS 9 support.
That said, I’m not iOS developer and so you’re probably right
